mackerel-plugin-mongodb/lib/mongodb.go

```diff
@@ -288,6 +324,7 @@ func Do() {
 	optPort := flag.String("port", "27017", "Port")
 	optUser := flag.String("username", "", "Username")
 	optPass := flag.String("password", os.Getenv("MONGODB_PASSWORD"), "Password")
+	optSource := flag.String("source","admin","authenticationDatabase")
```

Comment:
Is it correct that default value is **admin** in old version too?

I looked source of mgo, 	According to http://bazaar.launchpad.net/+branch/mgo/v2/view/head:/session.go#L343

```go
session.sourcedb = info.Source
if session.sourcedb == "" {
	session.sourcedb = info.Database
	if session.sourcedb == "" {
		session.sourcedb = "admin"
	}
}
```

I think empty string might be more compatible.

Reply:
thanks. modified default values to empty string.